lib/functions/function_plot_measures.R

```diff
@@ -0,0 +1,31 @@
+plot_measures <- function(
```

Comment:
You could start documenting the function like this:

```suggestion
#' Give the function a title
#'
#' @param data Describe data argument
#' @param title Describe title argument
#' @param x_label Describe x_label argument
#' @param y_label Describe y_label argument
#'
#' @return Describe what the function returns
plot_measures <- function(
```